src/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required (VERSION 2.8.11)
+project (TIGER)
+
+add_subdirectory(server)
+add_subdirectory(client)
+add_subdirectory(test)
+
+find_package(asio CONFIG REQUIRED)
+target_link_libraries(main PRIVATE asio asio::asio)
+
+find_package(Crow CONFIG REQUIRED)
+target_link_libraries(main PRIVATE Crow::Crow)
```

Comment:
Including crow should be good

Reply:
Cmake currently has no way to build crow though. I can add crow as a submodule, that way it will come with crow's cmake file too

Reply:
Having just the crow headers here would work if crow and asio are installed locally. However, this makes updating libraries and especially Continuous Integration difficult, let's keep libraries in the tree and build them alongside our own code.

Reply:
Looks like Asio is header only and needs to be installed, so let's do it that way.
I'll work on fixing the cmake setup.

Reply:
Alright understood

Reply:
So I should clone the crow repo inside of this one run cmake from within it?

Reply:
Yes, the entire source of the library should be included

Reply:
Actually I already have it partially set up, so I can take care of it
Let's discuss it in detail at the meeting